superset-frontend/src/chart/Chart.jsx

```diff
@@ -68,6 +68,8 @@ const propTypes = {
 };
 
 const BLANK = {};
+const NONEXISTENT_DATASET =
+  'The dataset associated with this chart no longer exists';
```

Comment:
Since this error is [translated in the backend](https://github.com/apache/superset/blob/be7065faf8e00ea0304b5380b2347f2ea9e4b39b/superset/views/utils.py#L252), I think we should wrap this in `t(...)` as well.